dists/android/build.gradle

```
@@ -54,8 +54,8 @@ android {
         // (p is patch/bugfix release number)
         // (b is build number, eg. re-release or beta testing was done on Play Store)
         // (a is a number indicating the target architecture (ABI)):
-        //     (1: arm-v7a, 2: arm64-v8a, 3: x86, 4: x86_64)
-        // eg. ScummVM 2.9.0 builds would have version codes: 2090001 - 2090004
+        //     (0: unspecified/fat, 1: arm-v7a, 2: arm64-v8a, 3: x86, 4: x86_64)
+        // eg. ScummVM 2.9.0 builds would have version codes: 2090000 - 2090004
```

This will need to be reverted too
```suggestion
        // eg. ScummVM 2.9.0 builds would have version codes: 2090001 - 2090004
```

---

Thanks, you are right.
I updated the version codes list below and adapted the example for ScummVM 2.10.